packages/manager/apps/dedicated/client/app/ip/ip/agoraOrder/ip-ip-agoraOrder.controller.js

```diff
@@ -148,65 +150,104 @@ angular.module('Module.ip.controllers').controller(
       return 'USA';
     }
 
+    loadPrivateCloudIpOffers(serviceName) {
+      return this.$http
+        .get(`/order/cartServiceOption/privateCloud/${serviceName}`)
+        .then((data) => get(data, 'data'))
+        .then((offers) =>
+          filter(offers, {
+            family: 'ip',
+          }),
+        )
+        .then((offers) => {
+          this.ipOffers = map(offers, (offer) => {
+            const price = head(offer.prices);
+            const maximumQuantity = get(price, 'maximumQuantity') || 1;
+            return {
+              ...offer,
+              productShortName: offer.productName,
```

Comment:
productShortName and productName are identical, why not use productName directly?

Reply:
because the legacy createOfferDto uses a productName and a productShortName so i wanted to keep those attributes despite those values being the same